.env.template

```diff
@@ -1,3 +1,9 @@
-BLOBS_URL=someserver.com/blob
-ROOM_ADDRESS=multiaddress
-ROOM_URL=https://someroomserver.com
+DB_PATH=/tmp/planetary-graphql
+
+ROOM_HOST=civic.love
+ROOM_PORT=8008
+ROOM_KEY=@uhI9kMjtjJSK4R6R5NGAm51e2iSC5D8Q354tGvbzBI4=.ed25519
```

Comment:
NOTE :fire:
I dropped ROOM_ADDRESS in favour of using these 3.
I think it's easier / safer?